src/components/ChangelogDialog/ChangelogDialog.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+import {block} from '../utils/cn';
+import {Dialog, DialogProps} from '../Dialog';
+import {Link} from '../Link';
+import {Icon} from '../Icon';
+import {ExternalLinkIcon} from '../icons/ExternalLinkIcon';
+import {ChangelogItem, OnStoryClick} from './types';
+import {Item} from './components/Item/Item';
+import i18n from './i18n';
+
+import './ChangelogDialog.scss';
+
+const b = block('changelog-dialog');
+
+export interface ChangelogDialogProps {
+    open: boolean;
+    title?: string;
+    fullListLink?: string;
+    items: ChangelogItem[];
+    disableBodyScrollLock?: boolean;
+    disableOutsideClick?: boolean;
+    onClose?: DialogProps['onClose'];
+    onStoryClick?: OnStoryClick;
+}
+
+export function ChangelogDialog({
+    open,
+    title = i18n('title'),
+    fullListLink,
+    items,
+    disableBodyScrollLock = true,
+    disableOutsideClick,
+    onClose,
+    onStoryClick,
+}: ChangelogDialogProps) {
+    const handleClose = React.useCallback<NonNullable<ChangelogDialogProps['onClose']>>(
+        (event, reason) => {
+            onClose?.(event, reason);
+        },
+        [onClose],
+    );
+
+    return (
+        <Dialog
+            className={b()}
+            open={open}
+            onClose={handleClose}
+            disableBodyScrollLock={disableBodyScrollLock}
+            disableOutsideClick={disableOutsideClick}
+        >
+            <Dialog.Header caption={title} />
```

Comment:
Let's use `h2` here.
```suggestion
            <Dialog.Header caption={<Text as="h2" variant="subheader-3">{title}</Text>} />
```

Reply:
It should be supported on `Dialog.Header` component level, isn't it?

Reply:
OK, let's do it in Dialog component in another PR.